nvflare/fuel/hci/client/api_spec.py

```diff
@@ -223,21 +209,36 @@ def check_command(self, command: str) -> CommandInfo:
         pass
 
 
-def service_address_changed_cb_signature(host: str, port: int, ssid: str):
-    pass
+class AdminConfigKey:
+    PROJECT_NAME = "project_name"
+    IDENTITY = "identity"
+    SERVER_IDENTITY = "server_identity"
+    CONNECTION_SCHEME = "scheme"
+    CONNECTION_SECURITY = "connection_security"
+    CLIENT_KEY = "client_key"
+    CLIENT_CERT = "client_cert"
+    CA_CERT = "ca_cert"
+    HOST = "host"
```

Comment:
We should remove host/port from admin config.

Reply:
No. How does the admin client know the host/port info without them?